crates/forge/tests/e2e/common/runner.rs

```diff
@@ -38,8 +38,26 @@ pub(crate) fn setup_package_with_file_patterns(
     file_patterns: &[&str],
 ) -> TempDir {
     let temp = tempdir_with_tool_versions().unwrap();
-    temp.copy_from(format!("tests/data/{package_name}"), file_patterns)
-        .unwrap();
+
+    let is_from_docs_listings = fs::read_dir("../../docs/listings")
+        .unwrap()
+        .map(|entry| entry.unwrap().file_name().into_string().unwrap())
+        .any(|entry| entry == package_name);
+
+    let package_path = if is_from_docs_listings {
+        format!("../../docs/listings/{package_name}",)
+    } else {
+        format!("tests/data/{package_name}",)
+    };
```

Comment:
What I meant when I said "inlining" was this 😅 
```suggestion
    let package_path = if is_from_docs_listings(...) {
        format!("../../docs/listings/{package_name}",)
    } else {
        format!("tests/data/{package_name}",)
    };
```

Not inlining the whole function

Reply:
Thne we can't inline it because `is_from_docs_listings` is used [here](https://github.com/foundry-rs/starknet-foundry/blob/dd4ede75a189ef2388b8b037660d5a6ec15d100c/crates/forge/tests/e2e/common/runner.rs#L89) as well.